### PR TITLE
Make changes to README and add Screenshot.

### PR DESCRIPTION
**Pull request by Marisa5280**

### CODE REVIEW

**Comment by Marisa5280** [comment]
this is what else I wrote but for some reason my git isnt recognizing the changes so it won't let me commit the completed readme.md
# RomCom  

### Abstract:
[//]: <> 
ROMCOM is a web app that can make generated romance novel covers, both random and customizable. These covers can be saved to the app and later deleted.

### Installation Instructions:
[//]: <> 
With the link to the GitHub Repository for this app, fork the repo and cloned to the user's local, and then run using 'open index.html'

### Preview of App:
[//]: <> 

### Context:
[//]: <> 
We had five days to finish this project. It took 4 days to complete. We are currently in week 2 of MOD 1.
### Contributors:
[//]: <> 
Marisa Wyatt https://github.com/Marisa5280
Jamie Caudill https://github.com/JamieCaudill
### Learning Goals:
[//]: <> 
-Write clean, DRY JavaScript
    -Build out functionality using functions that show trends toward SRP
    -Manipulate the page after it has loaded adding, removing, and updating elements on the DOM
-Explore the connection between HTML, CSS, and JavaScript
    -Practice reading, understanding, and using provided code
-Write code collaboratively
    -Document changes with atomic commits & thorough code reviews
    -Communicate, troubleshoot, and plan effectively as a team
    -Ensure all team members are able to be heard and contribute throughout the project

TECH used: GitHub, VScode, Command line (terminal), google and youtube for research.
### Wins + Challenges:
[//]: <> 
Wins: 
-We were able to try different types of collaboration and find what worked best.
-Combining what we found in research to make the code run!

Challenges:
-One of the biggest challenges was being able to figure out what to do when stuck. This project really helped to practice effective googling, and reaching out to our other cohort mates, rocks, and mentors for guidance. 
-Another challenge was being on different levels of understanding of the concepts used in this project, we learned that when the partner that has a better understanding navigates while the other drives it helps to deepen comprehension for both.

File: README.md [hunk 1]
```diff
@@ -23,13 +9,15 @@ ______________________________________________________
 
 ### Abstract:
 [//]: <> (Briefly describe what you built and its features. What problem is the app solving? How does this application solve that problem?)
+ROMCOM is a web app that can make generated romance novel covers, both random and customizable. These covers can be saved to the app and later deleted.
 
 ### Installation Instructions:
 [//]: <> (What steps does a person have to take to get your app cloned down and running?)
+With the link to the GitHub Repository this app can be forked and cloned to the user's local, and then run using 'open index.html'
```
Comment (JamieCaudill) [inline README.md:16]: We should probably detail this a little bit more in depth. A step by step list would look good.

File: README.md [hunk 2]
```diff
@@ -23,13 +9,15 @@ ______________________________________________________
...
 
 ### Preview of App:
 [//]: <> (Provide ONE gif or screenshot of your application - choose the "coolest" piece of functionality to show off.)
-
+![](../../Desktop/Screenshot%202023-04-01%20at%201.35.18%20PM.png)
```
Comment (JamieCaudill) [inline README.md:20]: Creative! I like it

**Review by JamieCaudill — COMMENTED** [review]
Looks good as a rough draft. There are a couple of things I would like to add before tomorrow night.

**Pull request merged**